questions/qa-headers-charset.en.html

```diff
@@ -80,37 +80,63 @@ <h2>The i18n Checker</h2>
     <p>The <a class="print" href="https://validator.w3.org/i18n-checker/">Internationalization Checker tool</a>, developed by the W3C, checks web pages for various internationalisation issues. It also has an information section that summarises key internationalization-related information about a page, such as character encoding and language declarations, etc. That section tells you whether an encoding declaration is used in the HTTP header, and if so, what is the encoding.</p>
     <p>The i18n checker tool is particularly useful, since it also shows you  other encoding declarations used in the document, and raises a flag if there are differences.</p>
   </section>
-  
-  
-  
-  
-  
+
+
+
+
+
   <section id="webbased">
 <h2>Use a Web-based service</h2>
     <p>There are several services that show you all the HTTP headers and the (HTML) source of the document returned from the server
       after you enter the address of the document you are interested in:</p>
     <ul>
-      <li><a class="print" href="http://www.delorie.com/web/headers.html">Delorie HTTP Header Viewer</a></li>
-      <li><a class="print" href="http://www.rexswain.com/httpview.html">Rex Swain's HTTP Viewer</a></li>
+      <li><a class="print" href="https://www.rexswain.com/httpview.html">Rex Swain's HTTP Viewer</a></li>
+      <li><a class="print" href="https://tools.keycdn.com/curl">HTTP Header Checker by KeyCDN</a></li>
     </ul>
     <p>Note: W3C has no relationship to any of these services.</p>
     <p>In the HTTP headers, look for the <code class="kw" translate="no">Content-Type</code> header, and in particular for the <code class="kw" translate="no">charset</code> parameter, e.g.</p>
     <blockquote><code>Content-Type: text/html; charset=utf-8</code></blockquote>
     <p>Note: The <code class="kw" translate="no">charset</code> parameter may not be present. This is okay if your document itself indicates its character
       encoding.</p>
   </section>
-  
+
+  <section id="devtools">
```

Comment:
We already discussed a better heading.

Btw, these days i use some conventions when formatting source code that help with maintenance.  If you'd like to do the same (up to you) i suggest:
1. leaving 5-6 empty lines before a section tag (makes it much easier to skip to or find section starts)
2. leaving one blank line after a heading
3. removing all line indentation, but adding a blank line before each of the following tags: p, ol, li (after the first), etc.

Reply:
Thanks for the suggestion. I usually use one blank line after a section tag, and I think that's enough for me.

(Some specs use [auto-generated section names in the comment](https://github.com/w3c/csswg-drafts/blob/f1b985bd66bdc0b73ad6e76468342014e76cfaf8/css-syntax-3/Overview.bs#L248-L256), but I think that's too much.)